searching.md

```diff
@@ -293,7 +293,7 @@ included even if their notes were added a long time ago.
 cards where the note text was added/edited in the last n days. Requires
 Anki 2.1.28+ / AnkiMobile 2.0.64+.
 
-## Recently answered
+## Recently answered (for the first time)
```

Comment:
(for the first time) only applies to introduced:, so perhaps it's a bit confusing to have this in the title?

Reply:
It was supposed to apply to both keywords, therefore the brackets. Does the new approach work for you?